### PR TITLE
Add album type 'EP'

### PR DESCRIPTION
**Pull request by XDjackieXD**
Spotify seemingly added the album type "EP" which crashes librespot when attempting to load album metadata for such an album (spotify:album:5UlvhrjweEYAHFvwyzuV6U for example).

### CODE REVIEW

File: protocol/proto/metadata.proto [hunk 1]
```diff
@@ -50,6 +50,7 @@ message Album {
         ALBUM = 0x1;
         SINGLE = 0x2;
         COMPILATION = 0x3;
+	EP = 0x4;
```
Comment (plietar) [inline protocol/proto/metadata.proto]: Indentation is wrong here

**Review by plietar — CHANGES_REQUESTED** [review]

**Review by plietar — APPROVED** [review]

**Comment by plietar** [comment]
Thanks !

File: protocol/proto/metadata.proto (hunk 1 above)
Comment (XDjackieXD) [reply protocol/proto/metadata.proto]: Sorry I did miss that the file uses space indentation everywhere

**Comment by sashahilton00** [comment]
Now that's what I call efficient :)

**Comment by mfonville** [comment]
The checksum is broken now, in `protocol/files.rs` the following diff should be applied:
```
@@ -4,7 +4,7 @@ pub const FILES : &'static [(&'static str, u32)] = &[
     ("proto/authentication.proto", 2098196376),
     ("proto/keyexchange.proto", 451735664),
     ("proto/mercury.proto", 709993906),
-    ("proto/metadata.proto", 3146937699),
+    ("proto/metadata.proto", 2474472423),
     ("proto/pubsub.proto", 2686584829),
     ("proto/spirc.proto", 3618770573),
 ];
```

**Comment by XDjackieXD** [comment]
Yeah sorry... forgot to recalc the checksums after fixing the indentation.

**Pull request merged**